docs/maker.md

```diff
@@ -0,0 +1,141 @@
+# Maker CLI Documentation
+
+The maker CLI is a command-line interface for interacting with the maker server in the Coinswap protocol. It allows you to manage your maker wallet, view balances, and perform various operations.
+
+## Overview
+
+The maker CLI provides functionality to:
+- Monitor wallet balances
+- View UTXOs
+- Generate new addresses
+- Send funds to external addresses
+- Get network information
+
+## Installation
+
+The maker CLI is part of the Coinswap project. To build from source:
+
+```bash:docs/maker-docs.md
+git clone https://github.com/citadel-tech/coinswap
+cd coinswap
+cargo build --release
+```
+
+## Commands
+
+### Basic Commands
+
+1. **Ping**
+   - Tests connectivity to the maker server
+   ```bash
+   maker-cli ping
+   ```
+
+2. **View UTXOs**
+   - List seed UTXOs: `maker-cli seed-utxo`
+   - List swap coin UTXOs: `maker-cli swap-utxo`
+   - List contract UTXOs: `maker-cli contract-utxo`
+   - List fidelity UTXOs: `maker-cli fidelity-utxo`
+
+3. **Check Balances**
+   - Seed balance: `maker-cli seed-balance`
+   - Swap coin balance: `maker-cli swap-balance`
+   - Contract balance: `maker-cli contract-balance`
+   - Fidelity balance: `maker-cli fidelity-balance`
+
+4. **Address Management**
+   - Generate new address: `maker-cli new-address`
+
+5. **Send Funds**
+   ```bash
+   maker-cli send-to-address --address <BITCOIN_ADDRESS> --amount <SATS> --fee <SATS>
+   ```
+
+6. **Network Information**
+   - Get Tor address: `maker-cli get-tor-address`
+   - Get data directory: `maker-cli get-data-dir`
+
+## Configuration
+
+The maker CLI uses a default data directory at `~/.coinswap/maker`. This directory contains:
+- Wallet data
+- Debug logs
+- Configuration files
```

Comment:
Show the content of the datadir folder and explain what each file contains. Show the content of the configuration file and explain each field.